packages/llama-cpp/llama-cpp-backend-opencl.subpackage.sh

```diff
@@ -2,5 +2,5 @@ TERMUX_SUBPKG_DESCRIPTION="OpenCL Backend for LLaMA.cpp"
 TERMUX_SUBPKG_RECOMMENDS="ocl-icd"
 TERMUX_SUBPKG_SUGGESTS="opencl-vendor-driver | clvk"
 TERMUX_SUBPKG_INCLUDE="
-lib/libggml-opencl.so
+bin/libggml-opencl.so
```

Comment:
Those are shared libraries, not executables, they should be in `$PREFIX/lib`.

Reply:
Correct, I suppose that's why @licy183 originally added the post installation hook to copy them, except the original ones remain in `$PREFIX/bin` and render the ones in `$PREFIX/lib` useless.

Reply:
That seems wrong.

Reply:
I don't know what you've done in this PR. I've patched the searched dir in https://github.com/termux/termux-packages/blob/master/packages/llama-cpp/0001-search-dirs.patch.

Reply:
The patch allows for the backend libraries in `lib\` to get loaded _in case_ the ones in `bin\` don't exist, which they do as they were never removed. You could swap the lines to change the priority but the subpackages will still be pointless since every backend gets loaded regardless.
The only way to load backends on a per subpackage basis is to either remove the original libraries from `bin\` or, as a more upstream friendly approach, split them into subpackages directly from `bin\` (which is what this PR was originally meant to do in 8234bf1).

Reply:
Many days ago these libraries is not installed by cmake. This commit should be reverted rather than install the libs to `$PREFIX/bin`. https://github.com/ggml-org/llama.cpp/commit/6eba72b71c677ce9aae33c422a6e67008137987a.

Reply:
Yeah, I didn't realize you were copying these libraries from **the build directory** to 'lib/'. This wasn't an issue until the upstream decided to add an install command to copy them to `bin/`.

I updated the PR to revert the upstream commit.